### PR TITLE
adds full hero

### PR DESCRIPTION
**Pull request by cutestsun**

### CODE REVIEW

File: src/partials/hero.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      </button>
+    </div>
+
+    <address class="location-address">
+      <a href="./" class="location-link link">
+        <svg class="location-icon" width="20" height="20">
+          <use href="./images/icons.svg#icon-location"></use>
+        </svg>
+        <span class="location-text"
+          >Україна, м.&nbspПолтава, вул.&nbspМаршала Бірюзова,&nbsp15</span
```
Comment (svoits) [inline src/partials/hero.html]: Change `&nbsp` to `&nbsp**;**`. You just forgot ';' 
Please fix it everywhere.

image

**Review by svoits — CHANGES_REQUESTED** [review]
please also edit icons in the same way as in the header.
and fix errors

File: src/partials/hero.html (hunk 1 above)
Comment (cutestsun) [reply src/partials/hero.html]: fixed the problem

File: src/partials/pages/hotel/hero.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
+<section class="section-hero hero-hotel">
+  <div class="container hero-container">
+    <h1 class="hero-title">Mimiho</h1>
+    <h2 class="hero-subtitle hero-hotel-subtitle">Готель</h2>
+    <p class="hero-text">Смак життя 24/7</p>
```
Comment (svoits) [inline src/partials/pages/hotel/hero.html]: there is no 'смак життя 24/7' in hotel.
please delete and change margin between MIMIHO and the buttons

**Review by svoits — CHANGES_REQUESTED** [review]
please edit hotel hero

File: src/partials/pages/hotel/hero.html (hunk 2 above)
Comment (cutestsun) [reply src/partials/pages/hotel/hero.html]: the problem was fixed

**Review by svoits — APPROVED** [review]

**Pull request merged**